roles/core/set_hostname/molecule/default/converge.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - name: Converge
   hosts: all
+  vars:
+    domain_name: bluebanquise.local
```

Comment:
Is `domain_name` a mandatory variable?

Reply:
Yes, due to ternary 😐 Ansible always check all variables inside the ternary, even if not used.

Reply:
This needs to be documented then.